loadtest/loadtest.go

```diff
@@ -245,6 +246,29 @@ func (lt *LoadTester) Status() *Status {
 	}
 }
 
+func (lt *LoadTester) InjectAction(action string) error {
+	var activeControllers []control.UserController
+
+	lt.mut.Lock()
+
+	if lt.status.State != Running {
+		return ErrNotRunning
+	}
```

Comment:
Looks like we may return with the mutex locked here. But with the timeout gone now, I think we may as well lock for the whole function to keep it simple.

Reply:
Good catch.  I removed the defensive copy, lock for whole method, and changed to a RLock since we are only reading `lt.status` and `lt.activeControllers`.